people_m.rb

```diff
@@ -0,0 +1,72 @@
+require_relative 'student'
+require_relative 'teacher'
+require_relative 'general_m'
+
+class PeopleMethod < GeneralMethod
+  attr_accessor :people
+
+  def initialize
+    super
+    @people = []
+  end
+
+  def list_all_people
+    check_empty_list(@people, 'people')
+
+    @people.each_with_index do |person, index|
+      puts "#{index}) [#{person.class}] Name: #{person.name}, ID: #{person.id}, Age: #{person.age}"
+    end
+  end
+
+  def create_student
+    age, name = student_d
+    Student.new(age, name)
+  end
+
+  def student_d
+    print 'Age: '
+    age = gets.chomp
+
+    print 'Name: '
+    name = gets.chomp
```

Comment:
- [ ] It would be nice if you use the below method in your code to take inputs and print the message instead of using `print` and `gets.chomp` a lot of times to make your code DRY and also it would make your methods Single responsible :+1: Kindly make this method in a file and import it into those files where you are using `print` and `gets.chomp` at the same time :+1:

```ruby
  def user_input(messages)
    inputs = []
    messages.each do |msg|
      print "#{msg}: "
      inputs.push(gets.chomp)
    end
    inputs
  end
```

Reply:
Thanks you for the feedback!